p2p/src/net/libp2p/behaviour/connection_manager/mod.rs

```diff
@@ -143,9 +143,7 @@ impl ConnectionManager {
                 self.add_event(ConnectionManagerEvent::Behaviour(
                     BehaviourEvent::ConnectionError {
                         address: connection.addr().clone(),
-                        error: P2pError::DialError(DialError::IoError(
-                            std::io::ErrorKind::ConnectionRefused,
-                        )),
+                        error: P2pError::DialError(DialError::ConnectionRefusedOrTimedOut),
```

Comment:
This makes me think that a rebase (as opposed to a merge) is necessary to avoid redoing what's being already done in master